include/circt/Dialect/OM/OMOpInterfaces.td

```diff
@@ -56,4 +56,20 @@ def ClassFieldLike : OpInterface<"ClassFieldLike"> {
   ];
 }
 
+def IntegerArithmeticInterface : OpInterface<"IntegerArithmeticOp"> {
```

Comment:
Can we mention this is an interface for binary arithmetic operation?

Reply:
Yes, good point.